spec/models/orchestration_stack_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "spec_helper"
+
+RSpec.describe OrchestrationStack, :type => :model do
```

Comment:
The `:type => :model` is not necessary...it's auto-tagged by rpsec-rails by just living in the spec/models directory.